.gitignore

```diff
@@ -1,5 +1,7 @@
 # Ignore Gradle project-specific cache directory
 .gradle
 
+.idea
```

Comment:
You will have to do this command.

https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore